roles/contrail_master/tasks/os_contrail_nested.yaml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Create the get IP file to convert hostname to ip
+  template:
+    src: getIp.j2
+    dest: /tmp/getIp.py
+    owner: root
+    mode: 0644
+  run_once: True
+
+- name: Copy the wait for POD script to master node
+  template:
+    src: wait_for_pod.j2
+    dest: /tmp/wait_for_pod.sh
+    owner: root
+    mode: 0777
+  run_once: true
+
+- name: Create the contrail nested template file
+  template:
+    src: contrail-nested.j2
+    dest: /tmp/contrail-nested.yaml
+    owner: root
+    mode: 0644
+  run_once: True
+
+- name: Create the contrail namespace template file
+  template:
+    src: contrail-namespace.j2
+    dest: /tmp/contrail-namespace.yaml
+    owner: root
+    mode: 0644
+  run_once: True
+
+- name: Create the contrail namespace
```

Comment:
These tasks seem to be common w/ `os_contrail_master.yaml` - lets move them to something like `os_contrail_common.yaml` and include it here?

Reply:
I would personally not want to go switch files to look into configurations in a common file. Lets keep this as is for now.

Reply:
Sure, its mostly cosmetic change for convenience